Sources/SwiftProtobuf/Google_Protobuf_Wrappers+Extensions.swift

```diff
@@ -48,15 +56,16 @@ extension Google_Protobuf_DoubleValue:
   }
 
   func encodedJSONString(options: JSONEncodingOptions) throws -> String {
-    var encoder = JSONEncoder()
-    encoder.putDoubleValue(value: value)
-    return encoder.stringResult
-  }
-
-  mutating func decodeJSON(from decoder: inout JSONDecoder) throws {
-    var v: WrappedType.BaseType?
-    try WrappedType.decodeSingular(value: &v, from: &decoder)
-    value = v ?? WrappedType.proto3DefaultValue
+    if value.isFinite {
```

Comment:
The encoding changes give me slight pause, because it means the knowledge about these are in two places.  But since they are pretty simple, I guess it's fine; and I don't immediately see any better ideas to share it without the conversion overhead.